boot.sh

```diff
@@ -3,6 +3,8 @@
 cd /home/shippable/reqProc
 mkdir -p logs
 
+cp -r $REQEXEC_PATH $REQEXEC_SRC_DIR
```

Comment:
Let's add checks to ensure that these ENVs are present. Also, with those checks add where they're expected to come from in comments as the names aren't very obvious.